src/overlays/actors/ovl_En_Heishi/z_en_heishi.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_en_heishi.h"
+#include "objects/object_sdn/object_sdn.h"
 
 #define FLAGS 0x00000009
```

Comment:
I'm on mobile, but these should be decompiled with their appropriate bitfields flags.

Reply:
What does this mean? The line that is highlighted for this comment is an include statement for the SDN object; did you highlight the wrong line?

Reply:
Correct me if I'm wrong @Kenix3 , but pretty sure he means to use the new ACTOR_FLAG defines for the FLAGS variables. In this case it would be 
```suggestion
#define FLAGS (ACTOR_FLAG_1 | ACTOR_FLAG_8)
```
note the order is for our style, i.e ascending order

Reply:
Oh, yeah that makes sense. Thanks!